HelpAssistHer/help-assist-her
# Time format

## Description
- The time input is editable so the time can be changed
- The time is in 12 hours (am/pm) format
- Clicking the left arrow decreases the time by 30 min
- Clicking the right arrow increases the time by 30 min

## Test Plan
- run yarn webpack
- go to http://localhost:4000/verification/pregnancy-resource-center
- click 'verify next resource' button
- scroll down to hours component
- click on arrows left or right
   - if left arrow is clicked and time is missing then it will auto-fill 9:00 am
   - if right arrow is clicked and time is missing then it will auto-fill 5:00 pm
   - now clicking left arrow will decrease time by 30 min
   - and clicking right arrow will increase the time by 30 min
   - when it will reach 12:00 it will change to am to pm if it was am before otherwise, it will change to am
